eventsources/sources/awssns/start.go

```diff
@@ -91,6 +122,15 @@ func (router *Router) HandleRoute(writer http.ResponseWriter, request *http.Requ
 		return
 	}
 
+	// SNS Signature Verification
+	if router.verifySNS {
```

Comment:
I don't see how `router.verifySNS` is set, it's always false.

Reply:
that's right, wanted to check if there is a way to pass this flag value during run time. For now, made changes to make it always true

Reply:
It looks like the signature verification needs extra http call (correct if I'm wrong), which means it has a cost to do that. So I think it would be better to expose it as a optional field (`validateSignature`) in the spec, and let the user determine if he/she would like do that.

Reply:
yes, a GET call is made to fetch the cert. Made changes to use optional field in spec